_posts/2024-09-24-sanders-middle-school.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  "4CSCC at Sanders Middle School"
+date:   2024-09-24
+categories: event
+author: Franklin Stewart
+---
+![Sanders Middle School](/images/2024-09-24-sanders-middle-school/20240924_SMSBus.png)
+Attending the Four Corners Computer Science Convening [see our blog from this event here](http://127.0.0.1:4000/event/2024/07/09/four-corners-cs-convening.html) gave 4CSCC an advantage in networking and reaching out to more schools on the Navajo Nation.   During these networking sessions, Franklin met Tyrone Joe, a computer science teacher at Sanders Middle School; he was also awarded the 2023 CS Teaching Excellence Award for the 4 Corners region from the Computer Science Teachers Association & Infosys.  Mr. Joe is involved with three cohorts: NAU’s Let’s Talk Code, U of A’s Natives Who Code, and Code.org’s CS Discoveries.
```

Comment:
```suggestion
Attending the Four Corners Computer Science Convening [see our blog from this event here](/event/2024/07/09/four-corners-cs-convening.html) gave 4CSCC an advantage in networking and reaching out to more schools on the Navajo Nation.   During these networking sessions, Franklin met Tyrone Joe, a computer science teacher at Sanders Middle School; he was also awarded the 2023 CS Teaching Excellence Award for the 4 Corners region from the Computer Science Teachers Association & Infosys.  Mr. Joe is involved with three cohorts: NAU’s Let’s Talk Code, U of A’s Natives Who Code, and Code.org’s CS Discoveries.
```

Reply:
The link you have here won't work as it's pointing to a local URL (so it'll work on your computer, but not on the live website). I think this version should work, but we'll fix once it goes live if not.